regions/core/core.py

```diff
@@ -305,6 +305,17 @@ class RegionMeta(dict):
 
     key_mapping = {'point': 'symbol', 'text': 'label'}
 
+    def __init__(self, dictionary=None, **kwargs):
+
+        if dictionary:
+            for key, val in dictionary.items():
+                self.__setitem__(key, val)
+        if len(kwargs) > 0:
+            for key, val in kwargs.items():
+                self.__setitem__(key, val)
+        else:
+            super(RegionMeta, self).__init__(**kwargs)
```

Comment:
Actually it is, `super(RegionMeta, self).__init__(dictionary, **kwargs)` , since `dictionary` arg has a default `None` value , I think it's not needed?

Reply:
but `super(RegionMeta, self)` is `dict`, no?  So we need to pass in keyword arguments that are valid for a dictionary.